template/web/app/buy-me-coffee/page.tsx

```diff
@@ -19,20 +19,21 @@ export default function BuyMeCoffeePage() {
     setIsMounted(true);
   }, []);
 
-  //  Fix hydration issues
   if (!isMounted) {
     return null;
   }
 
   return (
     <>
       <Header />
-      <Main>
-        <Banner pageName="Buy Me A Coffee" pageUrl="buy-me-coffee" />
-        <BuyMeCoffeeContractDemo />
-        <Guide />
-      </Main>
-      <Footer />
+      <div className="mt-[60px]">
```

Comment:
Should this be fixed in the layout file?

Reply:
I wasn't able to find a fix in the Header file, let me try the layout file 👍

Reply:
I had to add this spacing between the Header and Main component to fix the overlapping header issue. I took a look at the layout file and it's not clear to me how we would add this styling without having access to the problem components

Reply:
Agreed...this PR is getting big 😅

Reply:
Nice, moving that styling to `Main` worked!